### PR TITLE
SEO #62

### PR DESCRIPTION
**Pull request by imPGupta**
Added few keywords and fixes few codes.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -4,8 +4,16 @@
         
         <title>Doc2Pen|Contact</title>
         
-    <meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="keywords" content="doc2pen, Doc2Pen, Text-to-handwriting, text to handwriting converter online free, convert text to handwriting app, text to handwriting online free,
```
Comment (smaranjitghose) [inline contact.html:10]: 1) Please do not  repeat the same keywords like ```doc2pen``` and ```Doc2Pen```
2) I would not suggest using ```text to handwriting```
3) Have short keywords

**Comment by smaranjitghose** [comment]
Additionally please list out the meta tags you are using for the SEO, clearly in the PR description.

Try using the open graph and twitter card ones as well

**Comment by anushbhatia** [comment]
Use keyword Research tool to give the search volume of the keywords in an excel sheet pm slack or share on channel

File: contact.html (hunk 1 above)
Comment (imPGupta) [reply contact.html:10]: Sure, I'll do the above changes.

**Comment by smaranjitghose** [comment]
Please fix the merge conflicts as well

**Pull request closed without merging**